superset/examples/echarts_dashboard.py

```diff
@@ -113,138 +496,813 @@ def load_echarts_dashboard() -> None:
+    },
+    "type": "CHART"
+  },
+  "CHART-24": {
+    "children": [],
+    "parents": [
+      "ROOT_ID",
+      "TABS-TOP",
+      "TAB-TOP-2",
+      "ROW-9"
+    ],
+    "id": "CHART-24",
+    "meta": {
+      "chartId": 24,
+      "height": 50,
+      "sliceName": "Mixed Chart",
+      "width": 6
+    },
+    "type": "CHART"
+  },
+  "CHART-25": {
+    "children": [],
+    "parents": [
+      "ROOT_ID",
+      "TABS-TOP",
+      "TAB-TOP-2",
+      "ROW-9"
+    ],
+    "id": "CHART-25",
+    "meta": {
+      "chartId": 25,
+      "height": 50,
+      "sliceName": "Partition Chart",
+      "width": 6
+    },
+    "type": "CHART"
+  },
+  "CHART-26": {
+    "children": [],
+    "parents": [
+      "ROOT_ID",
+      "TABS-TOP",
+      "TAB-TOP-2",
+      "ROW-9"
+    ],
+    "id": "CHART-26",
+    "meta": {
+      "chartId": 26,
+      "height": 50,
+      "sliceName": "Radar Chart",
+      "width": 6
+    },
+    "type": "CHART"
+  },
+  "CHART-27": {
+    "children": [],
+    "parents": [
+      "ROOT_ID",
+      "TABS-TOP",
+      "TAB-TOP-2",
+      "ROW-10"
+    ],
+    "id": "CHART-27",
+    "meta": {
+      "chartId": 27,
+      "height": 50,
+      "sliceName": "Nightingale Chart",
+      "width": 6
+    },
+    "type": "CHART"
+  },
+  "CHART-28": {
+    "children": [],
+    "parents": [
+      "ROOT_ID",
+      "TABS-TOP",
+      "TAB-TOP-2",
+      "ROW-10"
+    ],
+    "id": "CHART-28",
+    "meta": {
+      "chartId": 28,
+      "height": 50,
+      "sliceName": "Sankey Chart",
+      "width": 6
+    },
+    "type": "CHART"
+  },
+  "CHART-29": {
+    "children": [],
+    "parents": [
+      "ROOT_ID",
+      "TABS-TOP",
+      "TAB-TOP-2",
+      "ROW-10"
+    ],
+    "id": "CHART-29",
+    "meta": {
+      "chartId": 29,
+      "height": 50,
+      "sliceName": "Sunburst Chart",
+      "width": 6
+    },
+    "type": "CHART"
+  },
+  "CHART-30": {
+    "children": [],
+    "parents": [
+      "ROOT_ID",
+      "TABS-TOP",
+      "TAB-TOP-2",
+      "ROW-11"
+    ],
+    "id": "CHART-30",
+    "meta": {
+      "chartId": 30,
+      "height": 50,
+      "sliceName": "Treemap Chart",
+      "width": 6
+    },
+    "type": "CHART"
+  },
+  "CHART-31": {
+    "children": [],
+    "parents": [
+      "ROOT_ID",
+      "TABS-TOP",
+      "TAB-TOP-2",
+      "ROW-11"
+    ],
+    "id": "CHART-31",
+    "meta": {
+      "chartId": 31,
+      "height": 50,
+      "sliceName": "Treemap V2 Chart",
+      "width": 6
+    },
+    "type": "CHART"
+  },
+  "CHART-32": {
+    "children": [],
+    "parents": [
+      "ROOT_ID",
+      "TABS-TOP",
+      "TAB-TOP-2",
+      "ROW-11"
+    ],
+    "id": "CHART-32",
+    "meta": {
+      "chartId": 32,
+      "height": 50,
+      "sliceName": "Word Cloud Chart",
+      "width": 6
+    },
+    "type": "CHART"
+  },
+  "GRID_ID": {
+    "children": [],
+    "id": "GRID_ID",
+    "type": "GRID"
+  },
+  "HEADER_ID": {
+    "id": "HEADER_ID",
+    "meta": {
+      "text": "ECharts Dashboard"
```

Comment:
Should we rename this everywhere to avoid confusion later?  It looks like the title is updated but maybe we should do the filenames and other references to "ECharts" too.